includes/sanitizers/class-amp-pwa-script-sanitizer.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * PWA Plugin Sanitizer
+ *
+ * @since 2.3
+ * @package AMP
+ */
+
+use AmpProject\AmpWP\ValidationExemption;
+
+/**
+ * Class AMP_PWA_Script_Sanitizer
+ *
+ * @since 2.3
+ * @internal
+ */
+class AMP_PWA_Script_Sanitizer extends AMP_Base_Sanitizer {
+
+	/**
+	 * Sanitize the AMP response for offline/500 error pages.
+	 *
+	 * @since 2.3
+	 */
+	public function sanitize() {
+		if (
+			! ( function_exists( 'is_offline' ) && is_offline() ) &&
+			! ( function_exists( 'is_500' ) && is_500() )
+		) {
+			return;
+		}
+
+		$scripts = $this->dom->xpath->query( '//script[@id="wp-navigation-request-properties" or @id="wp-offline-page-reload"]' );
```

Comment:
Just note that this depends on https://github.com/GoogleChromeLabs/pwa-wp/pull/793 which is not yet in a stable release of PWA.

Reply:
Fixed this by updating XPath to `'//script[ @id = "wp-navigation-request-properties" or ( @type = "module" and contains( text(), "checkNetworkAndReload()" ) ) ]'`